pym/gentoolkit/eclean/search.py

```diff
@@ -583,6 +586,8 @@ def findPackages(
         print(pp.error("Error: %s" % str(er)), file=sys.stderr)
         exit(1)
 
+    libc_deps = find_libc_deps(var_dbapi, True)
```

Comment:
It seems that strip_libc_deps works with `realized` either True or False here, but I noticed in depgraph it used False. I guess a possible benefit of using False is that find_libc_deps won't drop atoms here if there are no installed matches for any reason:
```python
        try:
            libc_pkgs.add(portdb.match(atom)[0])
        except IndexError:
            continue
```

Reply:
Yeah, I was unsure whether to pass `True` or `False` here. I'll change it to `False`.